samples/Microsoft.Bot.Builder.TestBot.Json/Samples/ToDoBot/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
```

Comment:
>configurations [](start = 5, length = 14)

example of an added declarative bot json launch configuration to attach the server running in the bot web service, listening on port 4712